lib/createConfig.js

```diff
@@ -3,5 +3,25 @@ const getBaseConfig = require('./getBaseConfig');
 
 module.exports = (commandName, configFragment = {}) => {
   const baseConfig = getBaseConfig(commandName);
-  return merge(baseConfig, configFragment);
+  const ignoreWarnings = {
+    ignoreWarnings: [
+      // Ignore warnings raised by source-map-loader.
+      // some third party packages may ship miss-configured sourcemaps, that interrupts the build
+      // See: https://github.com/facebook/create-react-app/discussions/11278#discussioncomment-1780169
+      /**
+       *
+       * @param {import('webpack').WebpackError} warning
+       * @returns {boolean}
+       */
+      function ignoreSourcemapsloaderWarnings(warning) {
+        return (
+          warning.module &&
+          warning.module.resource.includes('node_modules') &&
+          warning.details &&
+          warning.details.includes('source-map-loader')
+        );
+      },
+    ],
+  };
+  return merge(baseConfig, {...ignoreWarnings, ...configFragment});
```

Comment:
I'm not sure, this is the right place for this. I think you should move it to `config/webpack.common.config.js` or `config/webpack.prod.config.js`.

Reply:
@xitij2000 thanks for the review

what are the potential downsides of this?
we would not be able to see source maps for vendor packages that are in the node_modules

We are suppressing a warning, can we alternatively generate this sourcemap ourselves if the package doesn't provide it?
I tried generating the source-maps using rules but still got the same errors, best guess is some mis-configuration on some of the packages.

Of course we can move it to the webpack config files. 
I have counted 11 MFE's listed below that would be affected
```
frontend-app-ora-grading
frontend-app-profile
frontend-app-account
frontend-app-course-authoring
frontend-app-library-authoring
frontend-app-program-console
frontend-app-authn
frontend-app-gradebook
frontend-app-publisher
frontend-app-payment
frontend-app-learning
```
some of which do not have a webpack config file, what do we do for those?

Reply:
Instead of duplicating this, won't moving this to the common file fix the need for duplication.